pages/getting-started/first-steps-with-memgraph.mdx

```diff
@@ -9,22 +9,7 @@ import CodeSnippet from '/components/code-snippet/CodeSnippet'
 
 # First steps with Memgraph
 
-In this tutorial, you will learn how to install Memgraph Platform, connect to it
-using Memgraph Lab and run your first query. 
-
-A tutorial on this topic is also available as a video on Memgraph's YouTube
-channel:
-
-<br />
-
-<iframe
-  width="560"
-  height="315"
-  src="https://www.youtube.com/embed/6qYsAMisqQg"
-  frameBorder="0"
-  allow="autoplay; encrypted-media"
-  allowFullScreen
-></iframe>
+In this tutorial, you will learn how to install Memgraph and run your first query. 
```

Comment:
I get why you removed this link to YT video, but let's add it at the bottom saying: if you're using previous versions of Memgraph Platform (Memgraph Platform < 2.15), check this video for the instructions on how to connect. I would leave this for the first release, and remove it afterwards.

Reply:
Added.